client/js/upload.js

```diff
@@ -26,7 +23,7 @@ function initialize() {
 	instance.addEventListener("error", (event) => {
 		// Reset progressbar
 		$("#upload-progressbar").width(0);
-		$("#connection-error").addClass("shown").text(event.message);
+		vueApp.connectionError = event.message;
```

Comment:
Depending how temporary using the UI like this is, we may want to rename `connectionError`